search.go

```diff
@@ -15,6 +15,9 @@ import (
 	"github.com/koron/go-ssdp/internal/ssdplog"
 )
 
+// Seconds corresponds to a number of seconds.
+type Seconds = uint
+
```

Comment:
I can't accept this change.

This is because it would change the signature of the `Search` function.
Changing from an int to a uint-based type doesn't seem like a big deal,
but making a change to a function that's already public is unacceptable.

Reply:
Oh, sorry, I did not mean to change the type form int to uint. Because of the new type being an alias with `= int` for int (if we ignore the fact that I accidentally wrote uint) this change is actually backwards compatible but makes the code cleaner.